packages/@react-spectrum/dropzone/docs/DropZone.mdx

```diff
@@ -0,0 +1,338 @@
+      </DropZone>
+    </>
+  );
+}
+```
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show CSS</summary>
+
+```css
+.images{
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  border-radius: var(--spectrum-alias-border-radius-small);
+}
+
+.is-filled {
+  border-style: solid;
+}
+
+```
+</details>
+
+The `DraggableImage` component used above is defined below. See [useDrag](../react-aria/useDrag.html) for more details and documentation.
+
+<details>
+  <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
+
+```tsx example render=false export=true
+function DraggableImage() {
+  let {dragProps, isDragging} = useDrag({
+    getItems() {
+      return [
+        {
+          'image/jpeg': 'https://images.unsplash.com/photo-1528722828814-77b9b83aafb2?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8MTF8fHNwYWNlfGVufDB8fDB8fHww&auto=format&fit=crop&w=900&q=60'
+        }
+      ];
+    }
+  });
+
+  return (
+    <div
+      {...dragProps}
+      role="button"
+      tabIndex={0}
+      className={`draggable ${isDragging ? 'dragging' : ''}`} >
+      <img
+        width="150px"
+        height="100px"
+        alt="Cosmic Sky"
+        src="https://images.unsplash.com/photo-1528722828814-77b9b83aafb2?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxzZWFyY2h8MTF8fHNwYWNlfGVufDB8fDB8fHww&auto=format&fit=crop&w=900&q=60"/>
+    </div>
+  );
+}
+```
+</details>
+
+### Replace message
+
+When a drop zone is in a filled state and has an object dragged over it, a message will appear in front of the drop zone. By default, this message will say "Drop file to replace". However, users can choose to customize this message through the `replaceMessage` prop. This message should describe the interaction that will occur when the object is dropped. 
+
+```tsx example
+function Example() {
+  let [isFilled, setIsFilled] = React.useState(false);
+
+  return (
+    <>
+      <Draggable />
+      <DropZone
+        isFilled={isFilled}
+        maxWidth="size-3000"
+        replaceMessage="This is a custom message"
+        onDrop={() => setIsFilled(true)}>
+        <IllustratedMessage>
+          <Upload />
+          <Heading>
+            <Text slot="label">
+              Drag and drop here
+            </Text>
+          </Heading>
+        </IllustratedMessage>
+      </DropZone>
+    </>
+  );
+}
+```
+
+### Visual feedback
+
+A drop zone displays visual feedback to the user when a drag hovers over the drop target by passing the `getDropOpertation` function. If a drop target only supports data of specific types (e.g. images, videos, text, etc.), then it should implement the `getDropOperation` prop and return `cancel` for types that aren't supported. This will prevent visual feedback indicating that the drop target accepts the dragged data when this is not true. [Read more about getDropOperation.](../react-aria/useDrop.html#getdropoperation)
```

Comment:
```suggestion
A drop zone displays visual feedback to the user when a drag hovers over the drop target by passing the `getDropOperation` function. If a drop target only supports data of specific types (e.g. images, videos, text, etc.), then it should implement the `getDropOperation` prop and return `cancel` for types that aren't supported. This will prevent visual feedback indicating that the drop target accepts the dragged data when this is not true. [Read more about getDropOperation.](../react-aria/useDrop.html#getdropoperation)
```